internal/handlers/auth/user_auth_handler.go

```diff
@@ -0,0 +1,54 @@
+package auth
+
+import (
+	"fmt"
+	"github.com/antonioo83/license-server/config"
+	"github.com/antonioo83/license-server/internal/models"
+	"github.com/antonioo83/license-server/internal/repositories/interfaces"
+	"net/http"
+	"strings"
+)
+
+const Admin = 1
+const User = 2
```

Comment:
У гошки есть такая фишка как iota, вроде в курсе не было, но может где-то читал
```suggestion
const (
      Admin = iota +1
      User =  iota + 1
     )
```
https://go.dev/play/p/s_KSRAVuD0N